sound/soc/sof/sof-priv.h

```diff
@@ -319,6 +320,13 @@ struct snd_sof_pcm {
 	bool prepared[2]; /* PCM_PARAMS set successfully */
 };
 
+struct sof_led_control {
+	unsigned int use_leds;
+	unsigned int direction;
+	unsigned int led;
+	unsigned int led_value;
```

Comment:
can these all be just bits?

Reply:
ok maybe, I'm little bit scared of using bit fields in these structs and for mask option I would need to define masks, field sizes, so not sure it it is worth saving this space and adding more code. I remove 1 int anyway.